src/WebUI/locales/en.yml

```diff
@@ -1150,3 +1170,7 @@ leaderboard:
 
 rankTable:
   title: Rank table
+
+releaseNotes:
+  latest: Latest
+  showAllRelease: '+ {count} release | + {count} releases'
```

Comment:
Let's use patch notes instead of releases. It's what TW uses so players will be more familiar with that.